app/src/main/java/org/mifos/mobile/theme/Theme.kt

```diff
@@ -0,0 +1,46 @@
+package org.mifos.mobile.theme
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+private val LightThemeColors = lightColors(
```

Comment:
I am not sure where you got all these color values, you might think of using https://material-components.github.io/material-components-android-compose-theme-adapter/ as we are in the initial phase of integration.

Reply:
we are using a mifos ui library , I picked up some of the values from there and the remaing from a github repository